tesla_ble_mqtt/translations/en.yaml

```diff
@@ -20,6 +20,9 @@ configuration:
   debug:
     name: Debug Logging
     description: Print verbose messages to the log for debugging
+  temperature_unit_fahrenheit:
+    name: Temperature Unit; Fahrenheit if true otherwise Celsius
```

Comment:
I wonder, I think the "bool" will appear as a radio button. So maybe "active" instead of "true".
In fact you could also change the name to "Use Farenheit temperature in climate settings"

Reply:
Isn't "bool" a toggle switch like for debug?  Maybe you mean to write "toggle switch"?!

Reply:
Ok so... for debug we have `name: Debug Logging` so here we could have `name: Enable Fahrenheit temperature unit` - Let me know if that works for you as well.